### PR TITLE
FIX: Read velocities from file in r.terabin

### PR DESCRIPTION
**Pull request by danielhollas**
Unfortunately, `r.terabin` has been broken since Dec 2019 since this commit:

https://github.com/PHOTOX/ABIN/commit/42eec0b1fc604641d3720adde01054c7078778d5#diff-9be526330a9213578d1f49c8c02341eb1a4474cd3aa8de81cda0a1051417f7d0R128

It would ignore the velocity file specified at the top in variable `$VELOC_IN`.

@suchanj I am not sure how much is this script used and how much are people reading velocities from a file...

This bug would should have been caught earlier, since for simulations without velocity file we were executing ABIN lwith an empty `-v ` like this:
```sh
abin -i input -x geom -v
```
Unfortunately, due to the way we were parsing the command line, this was interpreted as an empty string, and because the default for the velocity file name _is_ an empty string, this was a valid invocation. :-( I am fixing this here and explicitly checking for empty arguments.

I hate BASH :cry:

### CODE REVIEW

**Comment by yoricklassmann** [comment]
@danielhollas can confirm that your fix works as intended, thanks a lot! Let's just hope this bug doesn't affect too many people.

File: utils/r.terabin [hunk 1]
```diff
@@ -211,7 +211,7 @@ ABIN_CMD="$ABINEXE -i $ABIN_IN -x $GEOM_IN"
 if [[ $USE_HYDRA_NAMESERVER = "true" ]];then
   ABIN_CMD="$ABIN_CMD -M $TC_PORT"
 fi
-if [[ -z $VELOC_IN ]];then
+if [[ -n $VELOC_IN ]];then
```
[the remark this answers is not in the document]
Comment (suchanj) [reply utils/r.terabin:214]: Ugh, ok not great, but I think a) most of the people were not going to the $PHOTOX for the script b) _tera_ was kinda underused for the last few years. Nevertheless, we should consider making ABIN 1.1 default for the older clusters to catch this error.

File: tests/CMDLINE/test.sh [hunk 2]
```diff
@@ -11,11 +11,25 @@ ABINEXE=$1
...
+mv ERROR ABIN_ERROR2
+
+$ABINEXE -x >> abin.out 2>&1 || true
+mv ERROR ABIN_ERROR3
+
+$ABINEXE -i >> abin.out 2>&1 || true
+mv ERROR ABIN_ERROR4
+
+$ABINEXE -M >> abin.out 2>&1 || true
+mv ERROR ABIN_ERROR5
```
Comment (suchanj) [inline tests/CMDLINE/test.sh:35]: Nice!

**Review by suchanj — APPROVED** [review]

File: utils/r.terabin (hunk 1 above)
[the remark this answers is not in the document]
Comment (suchanj) [reply utils/r.terabin:214]: Great! 💥

**Pull request merged**